src/types.rs

```diff
@@ -1073,6 +1189,16 @@ pub struct Cursor<T> {
 #[derive(Clone, Copy, Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub struct AttributeTag(pub u8);
 
+#[cfg(feature = "std")]
+impl quickcheck::Arbitrary for AttributeTag {
+    //todo it is not clear to me if random instances of these are useful in tests
+    fn arbitrary(g: &mut Gen) -> AttributeTag { AttributeTag(quickcheck::Arbitrary::arbitrary(g)) }
```

Comment:
I think you are right. We should only generate valid attributetags, valid in the sense that they can appear in valid attribute lists. That should only be 0-14 I think (the definitive list is in the rust-src/id/src/types.rs in concordium-base).

Reply:
Done